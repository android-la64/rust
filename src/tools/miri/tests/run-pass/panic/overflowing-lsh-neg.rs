// ignore-windows: Unwind panicking does not currently work on Windows
#![allow(exceeding_bitshifts)]
#![allow(const_err)]

fn main() {
    let _n = 2i64 << -1;
}
