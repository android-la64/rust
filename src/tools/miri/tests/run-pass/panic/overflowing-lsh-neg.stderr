thread 'main' panicked at 'attempt to shift left with overflow', $DIR/overflowing-lsh-neg.rs:6:14
